crates/torii/grpc/src/client.rs

```diff
@@ -40,18 +62,50 @@ pub struct WorldClient {
 
 impl WorldClient {
     #[cfg(not(target_arch = "wasm32"))]
-    pub async fn new<D>(dst: D, _world_address: Felt) -> Result<Self, Error>
-    where
-        D: TryInto<tonic::transport::Endpoint>,
-        D::Error: Into<Box<(dyn std::error::Error + Send + Sync + 'static)>>,
-    {
-        Ok(Self {
-            _world_address,
-            inner: world_client::WorldClient::connect(dst).await.map_err(Error::Transport)?,
-        })
+    pub async fn new(dst: String, _world_address: Felt) -> Result<Self, Error> {
```

Comment:
```suggestion
    pub async fn new(dst: String, world_address: Felt) -> Result<Self, Error> {
```
World address is actually used, we can remove the `_` at this occasion. :)